deploy/lint

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash -e
 
-BASE_DIR="$(dirname "$(dirname "$(readlink -e "$0")")")" # /esp
+# http://stackoverflow.com/a/4774063
+pushd `dirname $0` > /dev/null
```

Comment:
People do have spaces in their dev server path, actually, we've dealt with this before.